src/api/ledger/balance-sheet.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const _ = require('lodash');
+const utils = require('./utils');
+const validate = utils.common.validate;
+const composeAsync = utils.common.composeAsync;
+const convertErrors = utils.common.convertErrors;
+
+function formatBalances({balances, obligations, assets}) {
+  const result = {};
+
+  if (!_.isUndefined(balances)) {
+    result.balances = balances;
+  }
+  if (!_.isUndefined(obligations)) {
+    result.obligations = obligations;
+  }
+  if (!_.isUndefined(assets)) {
+    result.assets = assets;
+  }
+
+  return result;
+}
+
+function getBalanceSheetAsync(account, options, callback) {
+  validate.address(account);
+  validate.getBalanceSheetOptions(options);
+
+  const ledger = options.ledgerVersion || this.remote.getLedgerSequence();
```

Comment:
Should be called `ledgerVersion` (the name `ledger` would refer to an object that contains all the ledger data).